File: index/src/test/resources/runtime-helpers/java-private/Entry.java
package probe;
public class Entry {
  private String value() { return "probe.Used"; }
  public static void main(String[] args) throws Exception { Class.forName(new Entry().value()).getDeclaredConstructor().newInstance(); }
}
class Used { public Used() { System.out.println("USED"); } }
class Unused { public Unused() { System.out.println("UNUSED"); } }

File: experiments/runtime-returns/fixtures/kotlin_chain/Entry.kt
package probe

object Entry {
    @JvmStatic
    fun main(args: Array<String>) {
        Class.forName(name("Used")).getDeclaredConstructor().newInstance()
    }
}

class Used {
    init { println("USED") }
}

class Unused

private fun name(suffix: String): String = prefix() + suffix
private fun prefix(): String = "probe."
